### PR TITLE
Added tolerations, use mirror rancher/kube-rbac-proxy

### PR DESCRIPTION
**Pull request by rawmind0**

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -14,9 +14,6 @@ certificates:
 image:
   pullPolicy: IfNotPresent
   pullSecrets: []
-  # community image
-  #repository: k8s.gcr.io/multitenancy/externalip-webhook
-  #tag: v1.0.0
   repository: rancher/externalip-webhook
   # Defaults to .Chart.appVersion
   # tag: ""
```
Comment (cbron) [inline chart/values.yaml]: This still needs an actual tag. The scripts which grabs images for images.txt needs it.

File: chart/templates/_helpers.tpl [hunk 2]
```diff
@@ -26,6 +26,25 @@ heritage: {{.Release.Service }}
...
+{{- end -}}
+
+{{- define "linux-node-selector" -}}
+{{- if semverCompare "<1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+beta.kubernetes.io/os: linux
+{{- else -}}
+kubernetes.io/os: linux
+{{- end -}}
+{{- end -}}
+
```
Comment (cbron) [inline chart/templates/_helpers.tpl:43]: Are these used anywhere ?
* https://github.com/rancher/charts/blob/dev-v2.5-source/packages/rancher-backup/charts/templates/deployment.yaml#L46-L47
* https://github.com/rancher/charts/blob/dev-v2.5-source/packages/rancher-backup/charts/templates/deployment.yaml#L56
* https://github.com/rancher/charts/blob/dev-v2.5-source/packages/rancher-backup/charts/templates/_helpers.tpl#L12-L18

File: Dockerfile.dapper [hunk 3]
```diff
@@ -32,7 +32,7 @@ RUN curl -sL ${HELM_URL} | tar -xzf - && \
     HOME=/go/src/github.com/rancher/externalip-webhook/ CATTLE_HELM_UNITTEST_VERSION=${HELM_UNITTEST_VERSION} /usr/bin/helm plugin install ${HELM_UNITTEST_URL} \
```
Comment (cbron) [inline Dockerfile.dapper:32]: Does chart and app version just depend on the tag ?

**Review by cbron — CHANGES_REQUESTED** [review]

File: chart/values.yaml (hunk 1 above)
Comment (rawmind0) [reply chart/values.yaml]: Done

File: chart/templates/_helpers.tpl (hunk 2 above)
Comment (rawmind0) [reply chart/templates/_helpers.tpl:43]: Updated to use them

File: Dockerfile.dapper (hunk 3 above)
Comment (rawmind0) [reply Dockerfile.dapper:32]: yes both depends on the tag

**Review by cbron — APPROVED** [review]

**Pull request merged**